subprojects/docs/src/docs/userguide/releases/compatibility.adoc

```diff
@@ -19,8 +19,7 @@ The sections below describe Gradle's compatibility with several integrations.
 Versions not listed here may or may not work.
 
 == Java
-A Java version between 8 and 20 is required to execute Gradle.
-Java 21 and later versions are not yet supported.
```

Comment:
Can we keep this line but update to say "Java 22"?

Reply:
Sure.